### PR TITLE
[ARM-plugin] Init Azure CI for OpenVINO Android ARM plugin

### PR DESCRIPTION
**Pull request by allnes**

### CODE REVIEW

**Comment by azhogov** [comment]
CMake Error at inference-engine/thirdparty/movidius/XLink/XLink.cmake:36 (message):
  libusb is required

File: .ci/azure/android_arm64.yml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    path: openvino
+
+  - script: |
+      set -e
+      $(OPENVINO_REPO_DIR)/install_build_dependencies.sh
+      # Move into contrib install_build_dependencies.sh
+      sudo apt --assume-yes install scons crossbuild-essential-arm64 libprotoc-dev protobuf-compiler
+      # OpenCV should provide install_build_dependencies.sh as well
+      # Move into resources
+      git clone https://github.com/opencv/opencv.git --depth 1 $(OPENCV_REPO_DIR)
```
Comment (SDxKeeper) [inline .ci/azure/android_arm64.yml:78]: Please make opencv clone via checkout step https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema&viewFallbackFrom=vsts#checkout

**Review by azhogov — APPROVED** [review]

File: .ci/azure/android_arm64.yml [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    inputs:
+      cmakeArgs: >
+        -GNinja
+        -DVERBOSE_BUILD=ON
+        -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
+        -DCMAKE_TOOLCHAIN_FILE=$(ANDROID_TOOLS)/ndk-bundle/build/cmake/android.toolchain.cmake
+        -DANDROID_ABI=$(ANDROID_ABI_CONFIG)
+        -DANDROID_STL=c++_shared
+        -DANDROID_PLATFORM=$(ANDROID_SDK_VERSION)
+        -DOpenCV_DIR=$(BUILD_DIR_OPENCV)/install/sdk/native/jni
```
Comment (dkurt) [inline .ci/azure/android_arm64.yml:127]: Why do we need build OpenCV while there are published releases for Android? 

https://github.com/opencv/opencv/releases/download/4.5.5/opencv-4.5.5-android-sdk.zip

Comment (allnes) [reply .ci/azure/android_arm64.yml:127]: Thank you, I will correct it near time with other fixes

**Pull request merged**